tests/unit/test_bo.py

```diff
@@ -233,3 +233,32 @@ def test_gradient(self, acq_maxvar):
         threshold_grad_diff = 0.25
         assert leftover_dim_1 < threshold_grad_diff
         assert leftover_dim_2 < threshold_grad_diff
+
+
+class Test_RandMaxVar:
+    """Run a collection of tests for the RandMaxVar acquisition."""
+
+    def test_acq_bounds(self, acq_randmaxvar):
+        """Check if the acquisition is performed within the bounds.
+
+        Parameters
+        ----------
+        acq_randmaxvar : RandMaxVar
+            Acquisition method.
+
+        Notes
+        -----
+        - acq_randmaxvar acquires points from a 2-D Gaussian noise model.
+
+        """
+        bounds = acq_randmaxvar.model.bounds
+        batch_size = 2
+        n_it = 2
+
+        # Acquiring points.
+        for it in range(n_it):
+            batch_theta = acq_randmaxvar.acquire(n=batch_size, t=it)
+
+        # Checking if the acquired points are within the bounds.
+        assert np.all(batch_theta >= bounds[0][0])
+        assert np.all(batch_theta <= bounds[0][1])
```

Comment:
I think this is basically just testing that NUTS produces samples within bounds? (Which by the way seems to be untested atm, but anyway would be more appropriate, and faster, in that context.)

Reply:
That's correct. I use this test to check if the acquisition function does not crash. Though if we rename the test for its true purpose i.e. `test_acquisition`, what would we `assert`?

Reply:
Oh, ok, that's fine then. In principle you don't have to assert anything. ;)